src/common/substrait/src/df_logical.rs

```diff
@@ -59,20 +59,14 @@ impl SubstraitPlan for DFLogicalSubstraitConvertorDeprecated {
 
     async fn decode<B: Buf + Send>(
         &self,
-        message: B,
-        catalog_manager: CatalogManagerRef,
+        _message: B,
+        _catalog_list: Arc<dyn CatalogList>,
     ) -> Result<Self::Plan, Self::Error> {
-        let plan = Plan::decode(message).context(DecodeRelSnafu)?;
-        self.convert_plan(plan, catalog_manager).await
+        unimplemented!()
```

Comment:
> DFLogicalSubstraitConvertorDeprecated

I've discussed this with @waynexia , he insists that better keep this in case we need some functions in that file.

Reply:
I'll remove it after one or two weeks